### PR TITLE
SRCH-1063 - activerecord-validate_unique_child_attribute

### PR DESCRIPTION
**Pull request by peggles2**
Update gem to support Rails 5.2

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.3.5
+2.5.3
```
Comment (MothOnMars) [inline .ruby-version]: This should match the version in `.travis.yml`. Let's use the most recent 2.5 version, 2.5.7.

File: Gemfile.lock [hunk 2]
```diff
@@ -2,35 +2,35 @@ PATH
   remote: .
```
Comment (MothOnMars) [inline Gemfile.lock]: Per our discussion, remove `Gemfile.lock`. We should also add it to `.gitignore`.

**Review by MothOnMars — CHANGES_REQUESTED** [review]
A couple of tweaks. We should also update the "Compatibility" section of the README to reflect the removal of support for Rails 3 & 4.

File: .ruby-version (hunk 1 above)
Comment (peggles2) [reply .ruby-version]: ok I was trying to keep it consistent with our other apps.

**Review by PatKelsh — APPROVED** [review]
lgtm

**Pull request merged**